### PR TITLE
chore: remove enzyme update hmm tests

### PR DESCRIPTION
**Pull request by elanjm12**

### CODE REVIEW

File: src/js/hmm/Classes.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    accession: string;
+    gi: string;
+    name: string;
+    organism: string;
+};
+
+type families = {
+    family1: number;
+    family2: number;
+};
```
Comment (igboyes) [inline src/js/hmm/Classes.ts]:
```suggestion
type Families = {
    family1: number;
    family2: number;
};
```

File: src/tests/setupTests.jsx [hunk 2]
```diff
@@ -64,11 +55,21 @@ const renderWithRouter = (ui, state, history, createReducer) => {
...
 
+class ResizeObserver {
+    observe() {}
+    unobserve() {}
+}
+
+export const attachResizeObserver = () => {
+    window.ResizeObserver = ResizeObserver;
+};
+
```
Comment (igboyes) [inline src/tests/setupTests.jsx:66]: Didn't you already do this in the checkbox PR?

**Review by igboyes — CHANGES_REQUESTED** [review]

File: src/tests/setupTests.jsx (hunk 2 above)
Comment (elanjm12) [reply src/tests/setupTests.jsx:66]: This is the same code. I believe that all I did is move it below renderWithRouter.

Comment (igboyes) [reply src/tests/setupTests.jsx:66]: Sorry. That was a stupid question.

**Review by igboyes — APPROVED** [review]

**Pull request merged**